src/markdown-pages/docs/cli-command-reference/runs-get.md

```diff
@@ -28,6 +29,24 @@ Use the tables provided to view the options for filtering test results, and for
 | `--format raw` | The _raw_ format output is useful if you are writing scripts to report on multiple test runs programmatically. The output from `galasactl runs get` is returned in a form that makes it easy for scripting to digest the individual pieces of data available. | 
 
 
+
+## <a name="result"></a>More about the `--result` option
+
+You might want to filter the test results that are returned based on status. For example, you might choose to return only failed tests, so that you can quickly check if you need to investigate an issue. 
+
+The _galasactl runs get_ command now supports the `--result` option, so that test runs which completed with a specified status are returned. Runs which completed with a status that is not specified are not displayed.
+
```

Comment:
again replace 'status' with 'result'
as status is another term which could be misleading